workflow/main.rb

```diff
@@ -29,7 +38,7 @@ def emoji_item(emoji)
     query = Regexp.escape(ARGV.first)
 
     items = Emoji.all.map(&:name).grep(/#{query}/).each do |code|
-      fb.add_item(emoji_item(Emoji.find_by_alias(code)))
+      fb.add_item(emoji_item(Emoji.find_by_alias(code))) unless is_image_only(Emoji.find_by_alias(code))
```

Comment:
The only change I might suggest is capturing `Emoji.find_by_alias(code)` in a variable before this line, if only because I don't know how gemoji does search and I'm a tad worried it is a slow linear search through every emoji.